recipes/matio/all/patches/require-cmake-3.10.patch

```diff
@@ -0,0 +1,8 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,4 +1,4 @@
+-cmake_minimum_required(VERSION 3.7)
++cmake_minimum_required(VERSION 3.10)
+ 
+ if(POLICY CMP0074)
+     cmake_policy(SET CMP0074 NEW)
```

Comment:
```suggestion
     cmake_policy(SET CMP0074 NEW)
--- a/cmake/thirdParties.cmake
+++ b/cmake/thirdParties.cmake
@@ -29,8 +29,8 @@ if(MATIO_WITH_HDF5)
     endif()
 endif()
 
-if(HDF5_FOUND)
     set(HAVE_HDF5 1)
+if(0)
     add_library(HDF5::HDF5 INTERFACE IMPORTED)
     if(MATIO_USE_CONAN AND TARGET CONAN_PKG::hdf5)
         # target from Conan
```

Reply:
That should likely go to a separate file, right?

Reply:
Or rename the patch. It's up to you.

Reply:
Done.